src/DevChatter.Bot.Web/wwwroot/js/sprite.js

```diff
@@ -1,24 +1,119 @@
-function Sprite(imageSrc, x, y, direction) {
+function Sprite(imageSrc, width, height, x, y, direction, mod) {
```

Comment:
Don't forget to remove `width` and `height` here. You're not using these anymore.

Reply:
well that's embarrassing